src/org/lsst/ts/jenkins/components/Csc.groovy

```diff
@@ -290,5 +312,23 @@ def getBranchName(changeTarget, branchName) {
     return branch
 }
 
+def update_branches(pkg) {
```

Comment:
Instead of updating each branch individually, consider updating all the repos in `/home/saluser/repos/`... Something like this:

```
sh """
    source /home/saluser/.setup_dev.sh

    for repo in \$(ls /home/saluser/repos/)
    do
        cd /home/saluser/repos/\$repo
        /home/saluser/.checkout_repo.sh ${WORK_BRANCHES}
        git pull
    done
"""
```